subgraphs/abracadabra/schema.graphql

```diff
@@ -878,15 +878,9 @@ type Account @entity {
   " All positions that belong to this account "
   positions: [Position!]! @derivedFrom(field: "account")
 
-  " All open positions this account has "
-  openPositions: [Position!]!
```

Comment:
Was this part of the schema? Or was it added to debug?

Reply:
This was an addon the abra dev did